job-service-container/src/test/java/com/hpe/caf/services/job/api/JobServiceIT.java

```diff
@@ -76,6 +76,8 @@
 public class JobServiceIT {
 
     private static final Logger LOG = LoggerFactory.getLogger(JobServiceIT.class);
+    private static final String RABBIT_PROP_QUEUE_TYPE = "x-queue-type";
```

Comment:
Move these private static finals to
JobServiceConnectionUtil
and make them public